python/mujinplanningclient/realtimerobotplanningclient.py

```diff
@@ -386,1649 +740,3318 @@ def GetTransform(self, targetname, connectedBodyName='', linkName='', geometryNa
-        """Returns information about the inst objects and sensors that are a part of those inst objects.
+    def GetInstObjectAndSensorInfo(self, instobjectnames=None, sensornames=None, unit='mm', timeout=10, ignoreMissingObjects=None, dynamicEnvironmentState=None, debuglevel=None, **kwargs):
+        # type: (Optional[list[str]], Optional[list[str]], str, float, Optional[bool], Optional[types.GetInstObjectAndSensorInfoParametersDynamicEnvironmentState], Optional[int], Optional[Any]) -> Optional[Any]
+        """
+        Returns information about the inst objects and sensors that are a part of those inst objects.
 
         Args:
-            instobjectnames (list[str], optional):
-            sensornames (list[str], optional):
-            unit (str, optional): The unit of the given values. (Default: 'mm')
-            timeout (float, optional): Time in seconds after which the command is assumed to have failed. (Default: 10)
-            ignoreMissingObjects (bool, optional): If False, will raise an error if the object is not found in the scene. Default: True.
+            instobjectnames: (Default: None)
+            sensornames: (Default: None)
+            unit: The unit of the given values. (Default: 'mm')
+            timeout: Time in seconds after which the command is assumed to have failed. (Default: 10)
+            ignoreMissingObjects: If False, will raise an error if the object is not found in the scene. Default: True. (Default: None)
+            dynamicEnvironmentState: Dynamic environment state that allows the user to set/create objects in a particular state dynamically. (Default: None)
+            debuglevel: Sets the debug level for the planning logs. For development. 3=INFO, 4=DEBUG, 5=VERBOSE. (Default: None)
         """
         taskparameters = {
             'command': 'GetInstObjectAndSensorInfo',
             'unit': unit,
-        }
+        }  # type: dict[str, Any]
         if instobjectnames is not None:
             taskparameters['instobjectnames'] = instobjectnames
         if sensornames is not None:
             taskparameters['sensornames'] = sensornames
+        if ignoreMissingObjects is not None:
+            taskparameters['ignoreMissingObjects'] = ignoreMissingObjects
+        if dynamicEnvironmentState is not None:
+            taskparameters['dynamicEnvironmentState'] = dynamicEnvironmentState
+        if debuglevel is not None:
+            taskparameters['debuglevel'] = debuglevel
         taskparameters.update(kwargs)
         return self.ExecuteCommand(taskparameters, timeout=timeout)
 
-    def GetInstObjectInfoFromURI(self, instobjecturi=None, unit='mm', timeout=10, **kwargs):
-        """Opens a URI and returns info about the internal/external and geometry info from it.
+    def GetInstObjectInfoFromURI(self, objecturi=None, unit='mm', timeout=10, instobjectpose=None, dynamicEnvironmentState=None, debuglevel=None, **kwargs):
```

Comment:
parameter name changed

Reply:
Should be fixed now.

Reply:
https://github.com/mujin/mujinplanningclientpy/pull/45/commits/ac88d2b414c132b133e279f100e321e5a41ce213